src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/components/RequestStateFlow.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import {
+    useTooltipRef,
+    DoneIcon,
+    SyncIcon,
+    HistoryIcon,
+    CloseIcon,
+} from "@equinor/fusion-components";
+import classNames from "classnames";
+
+import * as styles from "./styles.less";
+import PersonnelRequest from '../../../../../../models/PersonnelRequest';
+
+type RequestStateProps = {
+    item: PersonnelRequest;
+}
+
+type RequestItemState = "approved" | "pending" | "rejected";
+
+type RequestItemProps = {
+    requestStatus: RequestItemState,
+    index: number,
+}
+
+const states = ["Created state", "Sumbitted state", "Approved state"]
+
+const getRequestStates = (request: PersonnelRequest): RequestItemState[] => {
+    const state = request.state;
+
+    switch (+state) {
+        case 0:
+            return ["approved", "pending"];
```

Comment:
I think we should return the same number of steps no matter what the state is. Otherwise it might look like different workflows for the request with different amount of steps